CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 
 ### Bug Fixes
 
+- Renamed `optIntoAsa` to `optInToAsa` to remain naming convention consistency across the project. 
```

Comment:
It's should be in ###Breaking changes I think

Reply:
you are right!